test/graphql_practice/content_test.exs

```diff
@@ -7,25 +7,21 @@ defmodule GraphqlPractice.ContentTest do
 
   @invalid_attrs %{description: nil, title: nil}
 
-  describe "list_uploads/0" do
-    setup [:user, :upload]
+  setup [:user, :upload]
 
+  describe "list_uploads/0" do
     test "returns all uploads", %{upload: upload} do
       assert Content.list_uploads() |> set_nil_user() == [upload] |> set_nil_user()
     end
   end
 
   describe "get_upload!/1" do
-    setup [:user, :upload]
-
     test "get_upload!/1 returns the upload with given id", %{upload: upload} do
       assert Content.get_upload!(upload.id) |> set_nil_user() == upload |> set_nil_user()
```

Comment:
This is a bit unreadable for me 

`set_nil_user(upload)` is more legible than using a single pipe

`assert Content.get_upload!(upload.id) |> set_nil_user()` is the more serious offender. I recommend piping all of them or none.. not some.